worker/baggageclaim/api/volume_server.go

```diff
@@ -495,6 +506,13 @@ func (vs *VolumeServer) StreamIn(w http.ResponseWriter, req *http.Request) {
 			return
 		}
 
+		var errExceedStreamLimit volume.ErrExceedStreamLimit
+		if errors.As(err, &errExceedStreamLimit) {
+			hLog.Error("exceeded-stream-limit", err)
```

Comment:
Any reason to use Error instead of Info? Seems like same kind of error like others here.

Reply:
Because I want to always log this error. Because this is a protection mechanism, it doesn't suppose to happen frequently, but we want to see all violations.